insights/athena/views_sql/vw_test_global_state_per_app.sql

```diff
@@ -86,7 +90,6 @@ v1_fhir_events AS (
       and request_method = 'GET'
       and path LIKE '/v1/fhir%'
       and response_code = 200
-      and app_name != 'new-relic'
```

Comment:
I noticed that this filter is removed from multiple where clauses, would this result in requests for app 'new-relic' being counted?

Reply:
@JFU-GIT For this per_app view, I removed those lines.  The new-relic app gets excluded later on. See the ON part at the end of the file. It gets excluded via line: 1254 as part of excluding the 4 internal apps.